app/seeders/dinosaur_seeder.rb

```diff
@@ -0,0 +1,28 @@
+class DinosaurSeeder
+  DINOSAURS = [
+    {
+        name: "Giganotosaurus",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.